index.js

```diff
@@ -36,6 +36,12 @@ var augmenters = [
         nodeInfo.trackInputChanges = false
       }
     }
+  ], [
+    'fsFacadeFlag', function (nodeInfo) {
```

Comment:
does this need to be tested?

Reply:
https://github.com/broccolijs/broccoli-node-info/blob/931215ccf85d866dbd953cedb8cf6c3a4238691c/test/node_info_test.js#L29
added